### ISSUE TITLE
Paddles not working

### ISSUE THREAD

**Issue by thrust26** [stella-emu/stella#575]
Paddles are not working with the latest uImage.

The default mapping maps the paddles to J0/ (left paddles ) and J1/ (right) + AX and AY. But the R77 reads them as AZ and A3. 

Did the axis change?

**Comment by thrust26** [comment]
not that fast

**Comment by sa666666** [comment]
Honestly, I'd rather fix this at the kernel level, so we don't unnecessarily have a bunch of IFDEF's for R77

**Comment by DirtyHairy** [comment]
That's the kernel mappings for the left paddles:

```
  input_set_capability(input_left, EV_ABS, ABS_X);
  input_set_abs_params(input_left, ABS_X, AXIS_MIN, AXIS_MAX, 0, 0);
  input_set_capability(input_left, EV_ABS, ABS_Y);
  input_set_abs_params(input_left, ABS_Y, AXIS_MIN, AXIS_MAX, 0, 0);
  input_set_capability(input_left, EV_ABS, ABS_RX);
  input_set_abs_params(input_left, ABS_RX, AXIS_MIN, AXIS_MAX, 0, 0);
  input_set_capability(input_left, EV_ABS, ABS_RY);
  input_set_abs_params(input_left, ABS_RY, AXIS_MIN, AXIS_MAX, 0, 0);
```

They have been not been changed. Isn't the static joymap in `SettingsR77.cxx` tailored to those mappings? Maybe the way the joymap is interpreted by stella has changed.

**Comment by sa666666** [comment]
OK, if that's the case, then we should probably fix it there instead.  I would like to minimize IFDEF when we can.

**Comment by sa666666** [comment]
In retrospect, of course the joymap string has changed.  With @thrust26 new remapping code, the string is not in the correct format at all (I think?).

**Comment by DirtyHairy** [comment]
Then the course is clear I think 😏

**Comment by thrust26** [comment]
Fixed by 3951fbf
